.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: scala
+
+scala:
+
+   - 2.10.0
+
+script: sbt test
```

Comment:
I just noticed that we have a `.travis` and a `.travis.yml`. As far as I know, we only need `.travis.yml`. Please take my comments from `.travis`, apply them to this file, and remove `.travis`.